server/bundler/bin/bundler.js

```diff
@@ -21,9 +21,12 @@ var webpackConfig = require( process.cwd() + '/webpack.config' ),
  * Variables
  */
 var start = new Date().getTime(),
-	bundleEnv = config( 'env' ),
 	outputOptions;
 
+if ( 'development' === config( 'env' ) ) {
```

Comment:
In the `Makefile`, the bundler is only run when `env != development`, here: https://github.com/Automattic/wp-calypso/blob/master/Makefile#L188

That (conditionals) is hard to replicate in NPM scripts, so I opted for always running the bundler ([here](https://github.com/Automattic/wp-calypso/pull/13536/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R156)) and exit early if `env == development`.